Update tables to include consensus cell types in report (#939)
AlexsLemonade/scpca-nf: templates/qc_report/celltypes_qc.rmd

  )
}

if (has_consensus) {
  glue::glue(

>     "\n\nConsensus cell types are obtained by looking for agreement between `SingleR` and `CellAssign` using an ontology-aware approach.

Reviewer: ```suggestion
    "\n\nConsensus cell types obtained based on agreement between `SingleR` and `CellAssign` using an ontology-aware approach.
```

If no common label is identified, no consensus cell type is assigned.
See the [ScPCA Portal documentation](https://scpca.readthedocs.io/en/stable/processing_information.html#cell-type-annotation) for a full description on how these cell type annotations are assigned.

> As these annotations are based on the output from `SingleR` and `CellAssign`, only the consensus cell type assignments will be displayed in the below tables and plots."

Reviewer: I'm suggesting this rephrase because this sentence reads a bit strangely when submitter annotations are present, given that consensus aren't the _only only_ ones being shown. I think gets the point across well enough?

```suggestion
`SingleR` and `CellAssign` annotation results are provided in the [supplementary cell type QC report](`r params$celltype_report`)."
```

  format_celltype_n_table()
```

```{r, eval = has_consensus}
knitr::asis_output('### Consensus cell type annotations\n

> In this table, cells labeled "Unknown cell type" are those which did not have agreement between the annotation from `SingleR` and `CellAssign`.

Reviewer: ```suggestion
In this table, cells labeled "Unknown cell type" are those whose `SingleR` and `CellAssign` annotations did not agree.
```
